lib/kafo/configuration.rb

```diff
@@ -57,10 +64,13 @@ def initialize(file, persist = true)
 
     def save_configuration(configuration)
       return true unless @persist
+
+      trimmed = configuration.reject { |key, value| !DEFAULT.key?(key) }
```

Comment:
Does it make sense to ignore nil values?
```suggestion
      trimmed = configuration.reject { |key, value| !DEFAULT.key?(key) || value.nil? }
```

Reply:
If we ignore nil, we should ignore empty arrays, empty hashes, empty strings. I'd rather just dump the configuration and it's current set of values for values that we persist to the file.